app/controllers/pdl.py

```diff
@@ -1291,6 +1372,61 @@ def generate_title_i_total_summary_response(title_id, start_year, end_year):
 
     return final_summary
 
+def generate_title_ii_total_summary_response(title_id, start_year, end_year):
```

Comment:
In the summary response, it would be good to include the summary of all individual programs (like in Title I) and not just the total Title-2 amount. Please add attributes for program name, total payment, and payment in percentages like below:
```
    {
        "titleName": "Title II: Conservation",
        "totalPaymentInDollars": abc,
        "programs": [
            {
                "programName": "Environmental Quality Incentives Program (EQIP)",
                "totalPaymentInDollars": xyz,
                "totalPaymentInPercentage": ab.cd
            },
            ...
        ]
    }
```

Reply:
Resolving the conversation as the change has been implemented in this PR.